TADashboard/src/main/java/utils/common/Common.java

```diff
@@ -10,40 +10,40 @@
 
 public class Common {
 
-	public static String convertImageToURI(String imagePath) {
-		System.setProperty("org.uncommons.reportng.escape-output", "false");
-		BufferedImage img;
-		File image = new File(imagePath);
-		try {
-			img = ImageIO.read(image);
-			ByteArrayOutputStream convert = new ByteArrayOutputStream();
-			ImageIO.write(img, "png", convert);
-			String data = DatatypeConverter
-					.printBase64Binary(convert.toByteArray());
-			return data;
-		} catch (IOException e1) {
-			e1.printStackTrace();
-		}
-		return null;
-	}
+    public static String convertImageToURI(String imagePath) {
```

Comment:
why do you reformat  this file?

Reply:
Sorry, I've developed a habit of reformatting all files before pushing to git.

Reply:
I have reverted it by master code. I don't why , it's still here. But my code is same as master branch now.